File: src/main/java/tw/racket/lang/psi/RacketElementType.java
package tw.racket.lang.psi;

import com.intellij.psi.tree.IElementType;
import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;
import tw.racket.lang.RacketLanguage;

public class RacketElementType extends IElementType {
    public RacketElementType(@NotNull @NonNls String debugName) {
        super(debugName, RacketLanguage.INSTANCE);
    }
}

File: src/main/kotlin/tw/racket/lang/psi/RacketElementType.kt
package tw.racket.lang.psi

import com.intellij.psi.tree.IElementType
import org.jetbrains.annotations.NonNls
import tw.racket.lang.RacketLanguage

class RacketElementType(@NonNls debugName: String) : IElementType(debugName, RacketLanguage)